Automattic/woocommerce-payments
# Update available payment methods to card only after disabling UPE

Fixes #4422 

#### Changes proposed in this Pull Request

Update available payment methods to card only after disabling UPE
> Were there other solutions you considered? 

I was planning on calling `invalidateResolutionForStoreSelector( 'getSettings' )` to keep a single source of truth, but we already manually set enabled payments to `[ 'card' ]` when disabling.

> Why did you choose to pursue this solution?

I found we have used the set data store method to set the enabled payment method to `card`.  To keep it consistent, I add `updateAvailabledPaymentMethodIds` to update the card to the available payment method

<hr>

#### Screenshots
*Before*
<img width="630" alt="before" src="https://user-images.githubusercontent.com/56378160/183528654-c07a0b90-5cf8-41f8-b3c6-f31916393512.png">

*After*
<img width="554" alt="after" src="https://user-images.githubusercontent.com/56378160/183528664-1b903b03-cd80-4900-90d2-e21cfe9cccc6.png">

#### Testing instructions
1. Enable UPE, add an additional payment method like Bancontact, and save changes
2. Disable UPE
3. Expect to see only the credit card method and the UPE setup banner

*

-------------------


**Post merge**
